.github/workflows/build-all-and-upload.yml

```
@@ -0,0 +1,27 @@
+name: Build all Erlang versions and upload to PackageCloud
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 8 * * 1-5' # 08:30 mon-fri
+
+jobs:
+  build-all-and-upload:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
```

This will always use the latest stable CRuby

```suggestion
          ruby-version: ruby
```